docs/source/use_cases/discover_public_data/index.rst

```diff
@@ -1,14 +1,19 @@
-Discover Public Data
+Discover Published Data
 ========================
+Here we list the growing collection of genomic data resident in Google Cloud Platform and shared with the community.  Some datasets are open-access, such as 1,000 Genomes, whereas other datasets are controlled-access.  Each dataset page describes the source of the data and how to *access the data* or *apply for access* as applicable for the dataset.
+
 
 .. toctree::
   :maxdepth: 1
 
-  google_genomics_public_data
```

Comment:
It would be nice if this were broken into two lists:

Genomic Data
  1000_genomes
  platinum_genomes
  reference_genomes
  mssng_data
  pgp_public_data
  supercentenarians

Annotations
  clinvar_annotations
  tute_genomics_public_data
  ucsc_annotations